src/Nventive.Persistence.Sample/Nventive.Persistence.Sample.Mobile/iOS/Main.iOS.cs

```diff
@@ -0,0 +1,15 @@
+using UIKit;
+
+namespace Nventive.Persistence.Sample
+{
+	public class EntryPoint
```

Comment:
I'm curious about the name of the class, was it added automatically as EntryPoint?

Reply:
Yeah, I used Uno.ProjectTemplates.Dotnet version 4.6.18.

Use dotnet new install Uno.ProjectTemplates.Dotnet::4.6.18 to try it out.